plugins/com.google.cloud.tools.eclipse.appengine.standard.java8/.project

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It didn't, but it forced leveraging the facets constraint framework for testing compatibility (which is a good thing) rather than introducing a separate feature flag and littering `isJava8()`-style checks everywhere.